src/com/mambu/apisdk/util/URLHelper.java

```diff
@@ -146,7 +147,10 @@ public String createUrlWithParams(String urlString, ParamsMap paramsMap) {
 	 */
 	public static String makeUrlWithParams(String urlString, ParamsMap paramsMap) {
 
-		return paramsMap != null ? urlString + DELIMITER + paramsMap.getURLString() : urlString;
+		String paramDelimiter = urlString.contains(QUESTION_MARK_DELIMITER) ?

```

Comment:
this logic is for urls that already contains question marks?

Reply:
yep, append fullDetails query param with "?" if is first query param or append it with "&" if there are other query params already appended